src/pages/Donate/DonateSupportUs.vue

```diff
@@ -9,7 +9,6 @@
 					:button-text="buttonText"
 					:data="donationValues"
 					:form-disclaimer="formDisclaimer"
-					:form-submit-analytics="formSubmitAnalytics"
```

Comment:
We use this on `src/pages/Donate/DonateFromMacroHero.vue` to differentiate the source of these donation entries...

Reply:
For the new analytics events the urls should not be present. It is very easy on looker to get the URL of where an event was fired. With the modified analytics event here, you can easily pull the data on looker to get all one-time donations from the donation form. If you wanted to to differentiate the donations that came from the DonateFromMacroHero you could filter that subset with events on URL `/donate/support-kiva`